dataeng/jobs/analytics/DatabaseExportCoursewareStudentmodule.groovy

```diff
@@ -11,6 +11,7 @@ class DatabaseExportCoursewareStudentmodule {
     public static def job = { dslFactory, allVars ->
         allVars.get('ENVIRONMENTS').each { environment, env_config ->
             dslFactory.job("database-export-courseware-studentmodule-$environment") {
+                description('It enables an Opsgenie Heartbeat, sets the expected duration and exports student module database by provisioning and terminating EMR.')
```

Comment:
Job which generates a raw SQL dump of courseware_studentmodule table and separates it into a different tsv file for each course